package.json

```diff
@@ -13,10 +13,13 @@
     "winston": "^3.2.1"
   },
   "devDependencies": {
+    "jest": "^26.0.1",
     "nodemon": "^2.0.2"
   },
   "scripts": {
     "start": "node index.js",
-    "dev": "nodemon index.js"
+    "dev": "nodemon index.js",
+    "jest": "jest clearCache && jest",
+    "jest-watch": "jest --clearCache && jest --watch"
```

Comment:
Can we name this from jest-watch to test-watch?

Reply:
I'll update this name when I update the name of the current `jest` task.

Reply:
The name for this task has been changed on #39 and has already been merged onto this branch.